### PR TITLE
Created JdbcEmailTemplateRepository to get emailTemplates by ID

### PR DESCRIPTION
**Pull request by fabriciodigennaro**
## Trello ticket
[Ticket](https://trello.com/c/pX8gWXsW/91-implement-jdbcemailtemplaterepository)

## Description of changes

- Implement JdbcEmailTemaplateRepository
- Implement Integration Test
- Fix unit tests
- increase the min coverage to 75%

### CODE REVIEW

File: src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java [hunk 1]
```diff
@@ -20,14 +21,16 @@ public SendEmailNotificationUseCase(
...
-        EmailNotification emailNotification = new EmailNotification(
-                userEmailAddress,
-                emailTemplate.getSubject(),
-                emailTemplate.getBody()
-        );
-        this.emailService.send(emailNotification);
-    }
 
-}
+        if (emailTemplate.isPresent()) {
```
Comment (solblack) [inline src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java:27]: i'm missing a test for the case it's not present

File: src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java [hunk 2]
```diff
@@ -20,14 +21,16 @@ public SendEmailNotificationUseCase(
     }
 
     public void execute(UUID userId, UUID templateId) {
```
Comment (solblack) [inline src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java:23]: i would return a result of the operation (success or errors)

**Review by solblack — APPROVED** [review]

File: src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java (hunk 1 above)
Comment (fabriciodigennaro) [reply src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java]: done

File: src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java (hunk 2 above)
Comment (fabriciodigennaro) [reply src/main/java/com/parkingapp/notificationservice/application/sendemailnotification/SendEmailNotificationUseCase.java]: changed 🆙

**Review by solblack — APPROVED** [review]

**Pull request merged**